warehouse/query-core/src/main/java/datawave/query/jexl/visitors/PushdownLargeFieldedListsVisitor.java

```diff
@@ -108,17 +116,20 @@ public Object visit(ASTOrNode node, Object data) {
         fields.addAll(rangeNodesByField.keySet());
         
         for (String field : fields) {
+            // if fields is not specified or the current field is in fields it can be reduced
+            boolean canReduce = (this.fields == null || this.fields.contains(field));
             
             Collection<JexlNode> eqNodes = eqNodesByField.get(field);
             Collection<JexlNode> rangeNodes = rangeNodesByField.get(field);
             
             // if "_ANYFIELD_" or "_NOFIELD_", then simply add the subset back into the children list
             // if past our threshold, then add a ExceededValueThresholdMarker with an OR of this subset to the children list
-            if (!Constants.ANY_FIELD.equals(field)
+            if (canReduce
+                            && !Constants.ANY_FIELD.equals(field)
```

Comment:
This is kind of ugly, can we put an expression per line and wrap with formatter:off/on?

Reply:
ok